src/energyplus/CMakeLists.txt

```diff
@@ -674,28 +668,30 @@ include("${PROJECT_SOURCE_DIR}/embedded/EmbedFiles.cmake")
 embed_files("${FILES}" "${EMBEDDED_PATHS}" OUTPUT energyplus)
 
 set(${target_name}_depends
-  openstudio_model
+  ${openstudio_model_depends}
 )
+set(${target_name}_depends ${${target_name}_depends} PARENT_SCOPE)
```

Comment:
Why do we need to export to parent_scope here?

Reply:
some other sibling targets are using this variable. for better or worse. I feel like that is questionable design but that is how it is. See for instance https://github.com/NREL/OpenStudio/blob/676874f7e1a13cda2ffdcb697a1d461cc23fddaa/src/sdd/CMakeLists.txt#L31

Yes it would be simpler and easier to link to the target (energyplus) and you would expect energyplus libs dependencies to be transitive, however jason found that is not the case with these object libraries. Certainly surprising behavior from cmake, but @lefticus researched this a lot so i have accepted it.